test/tests/fvkernels/mms/mat-advection.i

```diff
@@ -1,91 +0,0 @@
-a=1.1
-
-[GlobalParams]
-  advected_interp_method = 'average'
-[]
-
-[Mesh]
-  [./gen_mesh]
-    type = GeneratedMeshGenerator
-    dim = 1
-    xmin = -0.6
-    xmax = 0.6
-    nx = 2
-  [../]
-[]
-
-[Variables]
-  [./v]
-    family = MONOMIAL
-    order = CONSTANT
-    fv = true
-  [../]
-[]
-
-[FVKernels]
-  [./advection]
-    type = FVMatAdvection
```

Comment:
FVMatAdvection is still here, why did you remove the test?

Can we no longer do MMS because we cant specify BCs?

Reply:
I think my logic was that `FVMatAdvection` is tested at 3-4 other instances with Dirichlet Boundary conditions. The main difference is that this input had `FVMatAdvectionFunctionBC` which I removed since it entirely relied on boundary ghost cells by not exactly enforcing the DirichletBC.